taigaProject/src/taigaApi/util/metric_service.py

```diff
@@ -107,17 +108,12 @@ def calc_burndown_day_data(auth_token, milestone, attribute_key):
     }
     
     total_business_value = 0
```

Comment:
Can you please create a map for this variable? Similar to what you did for cycle time.

Reply:
`total_business_value = {
       "bv": 0
}`

Reply:
Done.